config/filter.d/apache-auth.conf

```diff
@@ -12,10 +12,10 @@ before = apache-common.conf
 
 failregex = ^%(_apache_error_client)s (AH(01797|01630): )?client denied by server configuration: (uri )?\S*(, referer: \S+)?\s*$
             ^%(_apache_error_client)s (AH01617: )?user .*? authentication failure for "\S*": Password Mismatch(, referer: \S+)?$
-            ^%(_apache_error_client)s (AH01618: )?user .*? not found(: )?\S*(, referer: \S+)?\s*$
-            ^%(_apache_error_client)s (AH01614: )?client used wrong authentication scheme: \S*(, referer: \S+)?\s*$
+            ^%(_apache_error_client)s (AH01618: )?user .*? not found(: )?.*(, referer: \S+)?\s*$
```

Comment:
As the referer is not what we are trying to catch, perhaps we could remove all the referer parts at the end of the rules, finishing them with a simple `.*`.
What we want to catch and which is important is at the beginning of each rule, so it would work in any case and would be easier :)